pyproject.toml

```diff
@@ -36,8 +36,7 @@ dependencies = [
   "requests",
   "rich ~= 13.0",
   "rfc8785 ~= 0.1.2",
-  # NOTE(dm): Under very active development, so strictly pinned.
```

Comment:
NB: Removed the `NOTE` here since this is now in beta and point releases won't be breaking.